site/specs/webrtc.yml

```diff
@@ -2421,6 +2520,201 @@ components:
+      properties:
+        dialogType:
+          $ref: '#/components/schemas/dialogType'
+        dialogStateData:
+          oneOf:
+            - $ref: '#/components/schemas/sipDialogRequestStateData'
+    createDialogResponse:
+      type: object
+      description: a Dialog is used for exchanging user-level state information
+      properties:
+        dialogId:
+          type: string
+          description: Unique id of the participant.
+          readOnly: true
+          example: 320e2af6-13ec-498d-8b51-daba52c37853
+        dialogStateData:
+          oneOf:
+            - $ref: '#/components/schemas/sipDialogResponseStateData'
+    updateDialogRequest:
+      oneOf:
+        - $ref: '#/components/schemas/sipUpdateDialogRequestBody'
+    sipDialogRequestStateData:
+      type: object
+      description: The state data associated with a Dialog - dependent on the dialogType
+      required:
+        - to
+        - from
+      properties:
+        to:
+          type: string
+          description: the destination telephone number in e.164 format
+          example: '+16136211234'
+        from:
+          type: string
+          description: >-
+            the caller's telephone number in e.164 format, or the string UNKNOWN
+            or RESTRICTED
+          example: '+16136211234'
+        userToUser:
+          type: string
+          description: >
+            Information that will be handed to the SIP client to be passed
+            towards the far end user. The formatting of the contents is
+            completely dependent on the ability of the  end system to parse and
+            understand the contents
+          example: >-
+            eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c;encoding=jwt
+    sipDialogResponseStateData:
+      type: object
+      description: >
+        The dialogStateData contains the information that represents the current
+        state of the dialog.  Depending on the characteristics and state of the
+        Dialog this information may be missing,  current, out of date, or
+        idempotent.  Please see the Text documentation about the use of Dialogs
+        for further information.
+      properties:
+        state:
+          type: string
+          description: the name of the state that the Dialog has entered
+          example: ANSWERED
+          enum:
+            - NEW
+            - CALLING
+            - ANSWERED
+            - RINGING
+            - TERMINATED
+        to:
+          type: string
+          description: the destination telephone number in e.164 format
+          example: '+16136211234'
+        from:
+          type: string
+          description: >-
+            the caller's telephone number in e.164 format, or the string UNKNOWN
+            or RESTRICTED
+          example: '+16136211234'
+        userToUser:
```

Comment:
Why would we want/need to include this value in the response and event payloads? It's potentially sensitive information and wouldn't serve any purpose in a response or callback

Reply:
Let's discuss 

The rationale was largely to enable a pattern.  In the case that we are working with the state data other than the name does not change, but that is not necessarily the case for all state changes.   If we are to comply with a consistent pattern we can...
- return minimal information like only the state name, and support at GET
- return "what's changed" which entails bookkeeping on our part, or
- return "current externally visible state at the time of response" 

I was favoring the latter for event notifications and application modification.